biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -818,6 +825,21 @@ subroutine UpdatePlantHydrLenVol(ccohort,site_hydr)
     real(r8) :: norm                         ! total root fraction used <1
     integer  :: nlevrhiz                     ! number of rhizosphere levels
 
+    ! added by Junyan May 29, 2020
+    real(r8) :: dbh                          ! the dbh of current cohort                                             [m]   
+    real(r8) :: dbh_0                        ! the dbh of the sappling at recuitment                                 [m]
+
+    real(r8) :: dbh_max                      ! the maximum dbh a PFT can have as observed                            [m]
+    real(r8) :: dbh_rev                      ! ratio, similar to RWC*   
+
+    real(r8) :: z_fr                         ! rooting depth of a cohort                                             [m]
```

Comment:
Why is this called z_fr?

Reply:
I'm thinking through this too.  "fr" must mean fine-root right?  The rooting depth should reflect the maximum depth of all roots (suberized/structural/transporting as well) right?
z is pretty common in refering to a vertical coordinate, maybe "depth" or something else might be more descriptive.  Although we use "z_" do describe the soil depths, which is also consistent with CLM/ELM.  Although, in those cases those are depth arrays of the soil layers.

Reply:
I get the 'z'. I was being throw off by my mental association of 'fr' with fraction.  Could it be called f_root?

Reply:
@rgknox , you are right fr stands for fine roots. Yes, the rooting depth should include both fine roots and coarse roots. Currently, hydro does not explicitly model the coarse roots fraction and depth, they are treated the same as fine roots. 
I think maybe just call it z_root ?

Reply:
z_root is good too.